HttpFs/HttpFs.fs

```diff
@@ -1060,16 +1060,25 @@ module Client =
         body             = response.GetResponseStream()
         luggage          = Some (upcast response) }
 
-  /// Sends the HTTP request and returns the full response as a Response record, asynchronously.
-  let getResponse request = job {
-    let webRequest, exec = toHttpWebRequest HttpFsState.empty request
-    do! exec
-    let! resp = getResponseNoException webRequest
-    let wrapped =
+  let tryGetResponse request =
```

Comment:
AFAIK this implementation always commits to the Alt: https://hopac.github.io/Hopac/Hopac.html#def:val%20Hopac.Alt.prepareJob – that's not the intention, is it?